apps/core/src/hooks/useGetNFTMeta.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useGetObject, useGetObjects } from './useGetObject';
+import { useMemo } from 'react';
+import { getObjectDisplay } from '@mysten/sui.js';
+
+const getNFTMeta = ({
+    name,
+    description,
+    creator,
+    image_url,
+    link,
+    project_url,
+}: Record<string, string>) => {
+    return {
+        name: name || null,
+        description: description || null,
+        imageUrl: image_url || null,
+        link: link || null,
+        projectUrl: project_url || null,
+        creator: creator || null,
+    };
+};
+
+export function useGetNFTMeta(objectID: string) {
+    const resp = useGetObject(objectID);
+    const nftMeta = useMemo(() => {
```

Comment:
Can you use a `select` on `useGetObject` instead? You can extend it to take a UseQuery config and pass that through.